controllers/storagecluster/storageclasses.go

```diff
@@ -22,7 +22,8 @@ import (
 )
 
 const (
-	storageClassSkippedError = "some StorageClasses were skipped while waiting for pre-requisites to be met"
+	storageClassSkippedError        = "some StorageClasses were skipped while waiting for pre-requisites to be met"
+	isDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-class"
```

Comment:
`defaultStorageClassAnnotation` lets rename it

Reply:
Copied the name from the Kubernets repo https://github.com/kubernetes/kubernetes/blob/dc8b57d8a75764e649d641ac77c1539a574d9aed/pkg/volume/util/storageclass.go#L32

Reply:
I am not sure why are they doing this, But it looks like this is a func variable instead of a const variable.

Reply:
Okay, updating

Reply:
Done